sig-apps/annual-report-2022.md

```diff
@@ -4,81 +4,73 @@
+    - [3140 - TimeZone support in CronJob](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/3140-TimeZone-support-in-CronJob) - v1.25
+    - [2307 - Job tracking without lingering Pods](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2307-job-tracking-without-lingering-pods) - v1.26
+    - [3329 - Retriable and non-retriable Pod failures for Jobs](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/3329-retriable-and-non-retriable-failures) - v1.26
   - stable:
     - [2214 - Indexed Job](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2214-indexed-job) - v1.24
     - [2232 - Suspend Job](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2232-suspend-jobs) - v1.24
+    - [1591 - Allow DaemonSets to surge during update like Deployments](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/1591-daemonset-surge/kep.yaml) - v1.25
     - [2599 - minReadySeconds for StatefulSets](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2599-minreadyseconds-for-statefulsets) - v1.25
 
-
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
+   - The mentoring cohort we've run in 2022 attracted a few new members, but we still need more.
 
 2. What metrics/community health stats does your group care about and/or measure?
 
-   -
-   -
-   -
+   - [Open untriaged issues and PRs](https://github.com/kubernetes/kubernetes/issues?q=is%3Aopen+is%3Aissue+label%3Aneeds-triage+label%3Asig%2Fapps)
 
 3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing
    to activities or programs that provide useful context or allow easy participation?
 
-   -
+   - It's up-to-date.
 
 4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide],
    does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
 
-   -
+   - No additional training than the recent mentoring cohort is planned, yet.
 
 5. Does the group have contributors from multiple companies/affiliations?
 
-   -
+   - [13 companies](https://k8s.devstats.cncf.io/d/8/company-statistics-by-repository-group?orgId=1&var-period=y&var-metric=contributions&var-repogroup_name=SIG%20Apps&var-repo_name=kubernetes%2Fkubernetes&var-companies=All&from=1609455600000&to=1639350000000) contributed code in 2022.
 
 6. Are there ways end users/companies can contribute that they currently are not?
    If one of those ways is more full time support, what would they work on and why?
 
-   -
-   -
+   - Increase the test coverage for all controllers, is probably the best way to help and start contributing to sig-apps owned controllers.
```

Comment:
Can you link out to a few of those places? just thinking follow up again

Reply:
I don't have an issue per se, but I've added a link to a proposal that Wojtek posted some time ago. This is usually something I'm personally checking with PRs touching controllers, to ensure we don't regress but rather every PR should increase the coverage for the package its touching.